cmd/kube-apiserver/app/options/options.go

```diff
@@ -153,6 +155,9 @@ func (s *ServerRunOptions) Flags() (fss cliflag.NamedFlagSets) {
 	fs.BoolVar(&s.AllowPrivileged, "allow-privileged", s.AllowPrivileged,
 		"If true, allow privileged containers. [default=false]")
 
+	fs.StringVar(&s.ShowHiddenMetrics, "show-hidden-metrics", s.ShowHiddenMetrics,
+		"The kube-apiserver version(x.y.z) for which you want to show hidden metrics. Only previous minor version is allowed.")
```

Comment:
Which version format should be expected?  actually, we don't care patch version, right?
- x.y.z
- x.y
- x.y.0

Reply:
I think "vX.Y" or "x.y". No need to use the patch version (changes at the wrong time) and it's definitely misleading to always want a ptach version of 0.

Reply:
I'd vote `x.y` personally.